statsmodels/discrete/discrete_model.py

```diff
@@ -4052,12 +4053,23 @@ def __init__(self, model, mlefit):
 
     def _maybe_convert_ynames_int(self, ynames):
         # see if they're integers
```

Comment:
this method doesn't use self, doesn't need to be a method.  could be collected with other name-standardizing utilities.

Reply:
Not going to do a major refactor to fix this one bug.